pytorch_lightning/trainer/connectors/checkpoint_connector.py

```diff
@@ -365,30 +389,6 @@ def dump_checkpoint(self, weights_only: bool = False) -> dict:
 
         return checkpoint
 
-    def hpc_load(self, checkpoint_path: str, on_gpu: bool):
```

Comment:
nit:

you could put the new function where it was so the diff is smaller and history is better preserved

Reply:
what if the intention was to move it? 

done :)